packages/base-driver/package.json

```diff
@@ -66,6 +66,7 @@
     "method-override": "3.0.0",
     "morgan": "1.10.0",
     "serve-favicon": "2.5.0",
+    "spdy": "4.0.2",
```

Comment:
Could be an optional deps? Perhaps this usage is limited so it should not block installation as possible failures

Reply:
yes, I can change that